spec/intro.dd

```diff
@@ -10,59 +10,58 @@ to create native libraries or executables.)
 $(P This document is the reference manual for the D Programming Language. For more information and
 other documents$(COMMA) see $(LINK2 https://dlang.org/, The D Language Website).)
 
-$(H2 Phases of Compilation)
+$(H2 Conceptual models)
```

Comment:
What is conceptual models? Can we put a description under here of what is meant by conceptual models?

Reply:
What are conceptual models?

Reply:
The idea is that the certain aspects are being introduced conceptually. This PR covers the process of translation, but I would like to add memory model, object model and execution model in follow up PRs.

Reply:
Yes this is abstruse. The old title made sense.

Reply:
Perhaps you are referring to the wrong title? It might be better to read the text rather than looking at the diff.

Reply:
Indeed sorry. But that doesn't change the general feeling I have about your initiative. I'm afraid that the direction taken could make the documentation "profane resistant", by introducing abstruse technical terms. But your lucky I'm not a desion maker ;)

Reply:
Removed the heading Conceptual Models as it seems to have caused confusion.